doc/redeemer.md

```diff
@@ -0,0 +1,56 @@
+# Ticket Redemption Service
+
+The Ticket Redemption Service allows Orchestrator nodes to redeem winning tickets on-chain using a separate ETH account on the Ticket Redemption Service. Orchestrator nodes no longer need to have an account funded with ETH to pay for transactions on the Ethereum network.
+
+It is responsible for redeeming winning tickets as well as pushing _max float_ updates about broadcasters back to its connected Orchestrators.
+
+_Max float_ is the guaranteed value an Orchestrator will be able from a Broadcaster's reserve. It accounts for the current reserve allocation from a Broadcaster to an Orchestrator as well as pending winning ticket redemptions. 
```

Comment:
"able from a Broadcaster's reserve" -> "able to claim from a Broadcaster's reserve"

Reply:
fixed in 9f594b4